app/workers/concerns/rate_limit_dump_worker.rb

```diff
@@ -12,8 +14,25 @@ module RateLimitDumpWorker
           "#{resource_type}_#{resource_id}_#{medium_id}_data_dump_worker"
         },
         enabled: ->(_resource_id, _resource_type, _medium_id, requester_id=nil) {
-          requester_id.blank? || !User.find(requester_id).admin?
+          congestion_enabled?(requester_id)
         }
       }
   end
+
+  module ClassMethods
+    def congestion_enabled?(requester_id)
+      # if the user is missing, should only happen via the rails console
+      return false if requester_id.blank?
+
+      if (skip_id_env = ENV.fetch('SKIP_DUMP_RATE_LIMIT_USER_IDS', nil))
```

Comment:
Should we parse this env var in an initialiser to keep all of them in the same place?

Reply:
I've extracted these - thanks for the suggestion 👍